envi/symstore/resolver.py

```diff
@@ -324,7 +324,9 @@ def getSymList(self):
         """
         Return a list of the symbols which are contained in this resolver.
         """
-        return [self.getSymByName(name) for name in self.symnames]
+        out = [self.getSymByName(name) for name in self.symobjsbyname]
+        out.extend([self.getSymByName(name) for name in self.symnames])
```

Comment:
actually, we have a test for symstore!  i went to write one and found it ;)  (`envi.tests.test_symstore`)
it seems all the pain we've run into have been vtrace's *use* of symstore?  and probably a oversight in coverage since it literally has two distinctive types of symbols, one that Win32 uses and one that ELF uses. <sigh>  we should probably put a todo item to coalesce them a little more (or at least make them normalize to one list internally).

although, i can see things in symstore's tests that could use some improvement.

oh mai, it's using raw asserts instead of unittest.asserts.... and dates back to the paleolithic era.

ugh... adding in a few new tests showed me another bug...

Reply:
buuuuuut, the test is probably good to stay.  just need to actually fix things so it passes.